### PR TITLE
Move Screen Up when keyboard opens in Login screen

### PR DESCRIPTION
**Pull request by mehul-gohappy**
Also added auto send OTP when 10 digits entered.

### CODE REVIEW

File: screens/loginScreen/LoginScreen.js [hunk 1]
```diff
@@ -115,6 +117,16 @@ class LoginScreen extends Component {
     var regexp = /^\+[0-9]?()[0-9](\s|\S)(\d[0-9]{8,16})$/;
     return regexp.test(this.state.phoneNumber);
   };
+  handlePhoneNumberInput = (text) => {
+    this.setState({ phoneNumber: text }, () => {
+      console.log("phone number inside state", this.state.phoneNumber);
```
Comment (Rakshit1411) [inline screens/loginScreen/LoginScreen.js:122]: remove this

File: screens/loginScreen/LoginScreen.js [hunk 2]
```diff
@@ -523,19 +536,18 @@ class LoginScreen extends Component {
...
+            behavior="height"
+            keyboardVerticalOffset={100}
+          >
             <PhoneInput
               style={styles.textInput}
               ref={this.state.phoneNumber}
               keyboardType="phone-pad"
               defaultCode="IN"
               layout="first"
-              onChangeText={(text) => {
```
Comment (Rakshit1411) [inline screens/loginScreen/LoginScreen.js:533]: why remove this ?

Comment (mehul-gohappy) [reply screens/loginScreen/LoginScreen.js:533]: we don't need that, firebase accepts only formatted texts, and it sets only those 10 digits in the state

File: screens/loginScreen/LoginScreen.js (hunk 1 above)
Comment (mehul-gohappy) [reply screens/loginScreen/LoginScreen.js:122]: console log or the whole function?

Comment (Rakshit1411) [reply screens/loginScreen/LoginScreen.js:122]: console log

File: screens/loginScreen/LoginScreen.js (hunk 2 above)
Comment (Rakshit1411) [reply screens/loginScreen/LoginScreen.js:533]: How is it related to firebase ?
This is just a function provided by PhoneInput library.

Comment (mehul-gohappy) [reply screens/loginScreen/LoginScreen.js:533]: If we type 8829865335 in the input
changeText will set 8829865335
but changeFormattedText will set +918829865335
which is the E164 format accepted by firebase auth

**Pull request merged**